lib/src/utf8.dart

```diff
@@ -26,7 +26,7 @@ class Utf8 extends Struct {
   static int strlen(Pointer<Utf8> string) {
     final Pointer<Uint8> array = string.cast<Uint8>();
     final Uint8List nativeString = array.asTypedList(_maxSize);
-    return nativeString.indexWhere((char) => char == 0);
```

Comment:
I'd also check if UTF-16 had the same issue.

Reply:
Utf16 is missing strlen(). Would it have to create a String to scan the code units? Something like `String.fromCharCodes(cstr.asTypedList()).length`?

Reply:
My bad, there is no strlen there because UTF-16 only goes from string to bytes, not the other way.
(I thought I saw two `strlen` implementations at some point. Maybe I was just looking at before and after a CL 🤭).